recipes/phenix/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "PHEnix" %}
+{% set version = "1-4-1a" %}
+{% set hash = "3153ebf851b0418f81afe0e3c1ec2b52a43067c134020d83a7d0faf351583741" %}
+
+package:
+  name: {{ name | lower }}
+  version: "{{ version | replace('-', '_') }}"
```

Comment:
can this be a dot?
Why is this such a strange version number :)

Reply:
I replaced "-" by ".",  but the version number I kept as same as the original release at here https://github.com/phe-bioinformatics/PHEnix/releases  :).